octodns/provider/cloudflare.py

```diff
@@ -537,12 +617,28 @@ def _gen_key(self, data):
                 '{precision_horz}',
                 '{precision_vert}')
             return ' '.join(loc).format(**data)
+        elif _type == 'URLFWD':
+            uri = data['targets'][0]['constraint']['value']
+            uri = '//' + uri if not uri.startswith('http') else uri
+            parsed_uri = urlsplit(uri)
+            ret = {}
+            ret.update(data['actions'][0]['value'])
+            ret.update({'name': parsed_uri.netloc, 'path': parsed_uri.path})
+            urlfwd = (
+                '{name}',
+                '{path}',
+                '{url}',
+                '{status_code}')
+            return ' '.join(urlfwd).format(**ret)
```

Comment:
If I'm reading this correctly I think it can be simplified to:

```suggestion
            return '{name} {path} {url} {status_code}'.format(name=parsed_uri.netloc, 
                                                              path=parsed_uri.path, 
                                                              **data['actions'][0]['value'])
```